src/script/conversation/ConversationLabelRepository.ts

```diff
@@ -121,22 +124,39 @@ export class ConversationLabelRepository extends TypedEventTarget<{type: 'conver
         type,
       }),
     );
+
     this.labels(labels);
   };
 
   readonly saveLabels = () => {
-    this.propertiesService.putPropertiesByKey(propertiesKey, this.marshal());
+    const values = this.marshal();
+    void this.propertiesService.putPropertiesByKey(propertiesKey, values);
+    this.persistValues();
   };
 
   loadLabels = async () => {
     try {
+      const values = localStorage.getItem(ConversationLabelRepository.LocalStorageKey);
```

Comment:
A minor suggestion, renaming it to something meaningful for example: `conversationLabel` or `conversationLabelJson`